core/src/main/resources/org/apache/calcite/runtime/CalciteResource.properties

```diff
@@ -336,4 +336,6 @@ InvalidPartitionKeys=Only tables with set semantics may be partitioned. Invalid
 InvalidOrderBy=Only tables with set semantics may be ordered. Invalid ORDER BY clause in the {0,number,#}-th operand of table function ''{1}''
 MultipleRowSemanticsTables=A table function at most has one input table with row semantics. Table function ''{0}'' has multiple input tables with row semantics
 NoOperator=No operator for ''{0}'' with kind: ''{1}'', syntax: ''{2}'' during JSON deserialization
+FromNotZero=Invalid input for position function: from operand value must not be zero
```

Comment:
s/position/POSITION/

Other error messages have the function in upper case. It reinforces that it's a function name.